explorer/src/components/WorldMap.tsx

```diff
@@ -18,21 +16,25 @@ type MapProps = {
 
 export const WorldMap: React.FC<MapProps> = ({ title, countryData, userLocation, loading }) => {
 
+  const [colorScale, setColorScale] = React.useState();
   const [hasNoContent, setHasNoContent] = React.useState<boolean>(true);
   const [tooltipContent, setTooltipContent] = React.useState<string | null>(null);
   const { mode } = React.useContext(MainContext);
 
-  const colorScale: any = scaleLinear()
-    .domain([0, 1200])
-    // @ts-ignore
-    .range(mode === 'dark' ? ['#ffedea', '#ff5233'] : ['orange', 'red']);
-
-
   React.useEffect(() => {
     if (userLocation || countryData) {
       setHasNoContent(false);
     }
+    if (countryData?.data) {
+      const heighestNumberOfNodes: number = countryData.data.reduce((a, b) => a.nodes > b.nodes ? a : b).nodes;
+      const cs: any = scaleLinear()
+        .domain([0, heighestNumberOfNodes])
+        // @ts-ignore
+        .range(mode === 'dark' ? ['#FFD278', '#F20041'] : ['orange', 'red']);
```

Comment:
I have to be a bit suspicious of a bit of code that has `any` and `ts-ignore` in it!

It seems like something that you're using doesn't have proper typings?

Reply:
Yep, bit of googling necessary there. thanks. sorted.